monitoring/mongodb/dashboard.py

```diff
@@ -0,0 +1,1359 @@
+            expr=f'sum by(type) (\n  rate(mongodb_mongod_op_latencies_latency_total{{job=~"{Jobs.shard_pattern}"}}[$__rate_interval])\n) /\nsum by(type) (\n  rate(mongodb_mongod_op_latencies_ops_total{{job=~"{Jobs.shard_pattern}"}}[$__rate_interval])\n)',
+            legendFormat="{{type}}",
+        )
+    ],
+    unit="µs",
+    thresholds=THRESHOLDS["latency"],
+    lineInterpolation="smooth",
+    fillOpacity=20,
+    showPoints="auto",
+    description="Average latency of operations (classified by read, write, or (other) command)",
+)
+
+replication_lag = mongodb_timeseries(
+    "Replication Lag",
+    [
+        Target(
+            expr=MongoDBMetrics.replication_lag(),
+            legendFormat="{{pod}} <- {{name}}",
+        )
+    ],
+    unit=UNITS.SECONDS,
+    thresholds=THRESHOLDS["replication_lag"],
+    lineInterpolation="smooth",
+    lineWidth=2,
+    fillOpacity=20,
+    showPoints="always",
+    decimals=2,
+    valueMin=0,
+    description="MongoDB replication lag occurs when the secondary node cannot replicate data fast enough to keep up with the rate that data is being written to the primary node.",
+)
+
+flow_control = mongodb_timeseries(
+    "Flow Control",
+    [
+        Target(
+            expr=f'max(rate(mongodb_ss_flowControl_isLaggedCount{{job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}}[$__rate_interval]))',
+            legendFormat="fc_count",
+        ),
+        Target(
+            expr=f'max(rate(mongodb_ss_flowControl_isLaggedTimeMicros{{job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}}[$__rate_interval]))',
+            legendFormat="fc_time",
+        ),
+    ],
+    unit="µs",
+    lineWidth=2,
+    fillOpacity=20,
+    showPoints="never",
+    decimals=2,
+    valueMin=0,
+    description="Flow Control is a mechanism that helps to prevent the primary node from being overwhelmed by the number of operations it needs to process. It is used to control the rate at which the primary node can write to the secondary node.",
+)
+
+# Performance Metrics Panels
+query_exec_times = mongodb_timeseries(
+    "Query execution times",
+    [
+        Target(
+            expr=f'avg by (op_type) (rate(mongodb_ss_opLatencies_latency{{job=~"{Jobs.shard_pattern}"}}[$__rate_interval]) / (rate(mongodb_ss_opLatencies_ops{{job=~"{Jobs.shard_pattern}"}}[$__rate_interval]) > 0))',
+            legendFormat="{{op_type}}",
+        )
+    ],
+    unit="µs",
+    thresholds=THRESHOLDS["query_exec"],
+    lineInterpolation="smooth",
+    lineWidth=2,
+    fillOpacity=20,
+    showPoints="never",
+    decimals=2,
+    valueMin=0,
+    description="Average latency of operations (classified by read, write, or (other) command)",
+)
+
+queued_ops = mongodb_timeseries(
+    "Queued Operations",
+    [
+        Target(
+            expr=f'max by (type) (mongodb_mongod_global_lock_current_queue{{job=~"{Jobs.shard_pattern}", rs_nm=~"$rs_nm"}})',
+            legendFormat="{{type}}",
+        )
+    ],
+    unit=UNITS.OPS_PER_SEC,
```

Comment:
looking at the metrics (`mongodb_mongod_global_lock_current_queue`), it seems weird that the unit would be `ops/sec`, as it looks (from the name "current") and is used (wihtout rate) like a gauge... Is the unit wrong, and should just the a number?